packages/firestore/test/unit/model/field_value.test.ts

```diff
@@ -304,15 +319,31 @@ describe('FieldValue', () => {
     const objValue = wrapObject({ foo: { bar: 'bar-value' }, a: 1 });
 
     const expected = { foo: { bar: 'bar-value' }, a: 1 };
-    const objValue2 = objValue.delete(field('foo.baz'));
-    const objValue3 = objValue.delete(field('foo.bar.baz'));
-    const objValue4 = objValue.delete(field('a.b'));
+    const objValue2 = deleteField(objValue, 'foo.baz');
+    const objValue3 = deleteField(objValue, 'foo.bar.baz');
+    const objValue4 = deleteField(objValue, 'a.b');
     expect(objValue.value()).to.deep.equal(expected);
     expect(objValue2.value()).to.deep.equal(expected);
     expect(objValue3.value()).to.deep.equal(expected);
     expect(objValue4.value()).to.deep.equal(expected);
   });
 
+  it('can delete multiple fields', () => {
```

Comment:
Optional: add a test showing it's possible to set and immediately delete fields? (e.g., set `a`, `b`, `c`, delete `b`, check that the end result only has `a`, `c`)

Reply:
Added a test.